src/harness/reference_models/interference/interference.py

```diff
@@ -0,0 +1,648 @@
+
+  # Get the total antenna gain by summing the antenna gains from CBSD to ESC
+  # and ESC to CBSD
+  effective_ant_gain = ant_gain + esc_ant_gain
+
+  # Compute the interference value for ESC entity
+  eirp = getEffectiveSystemEirp(max_eirp, cbsd_grant.antenna_gain,effective_ant_gain)
+
+  interference = eirp - db_loss - IN_BAND_INSERTION_LOSS
+  return interference
+
+
+def computeInterferenceFssCochannel(cbsd_grant, constraint, fss_info, max_eirp):
+  """Compute interference grant causes to a FSS protection point
+  
+  Routine to compute interference neighborhood grant causes to FSS protection 
+  point for co-channel passband
+  Args:
+    cbsd_grant: a namedtuple of type CbsdGrantInformation
+    constraint: protection constraint of type ProtectionConstraint
+    fss_info: contains information on antenna height the tangent 
+              and perpendicular components.
+    max_eirp: The maximum EIRP allocated to the grant during IAP procedure
+  Returns:
+    interference: interference contribution(dBm)
+  """
+
+  # Get the propagation loss and incident angles for FSS entity_type
+  db_loss, incidence_angles, _ = wf_itm.CalcItmPropagationLoss(cbsd_grant.latitude, 
+                                   cbsd_grant.longitude, cbsd_grant.height_agl, 
+                                   constraint.latitude, constraint.longitude, 
+                                   fss_info.height_agl, cbsd_grant.indoor_deployment, 
+                                   reliability=-1, freq_mhz=FREQ_PROP_MODEL_MHZ)
+
+  # Compute CBSD antenna gain in the direction of protection point
+  ant_gain = antenna.GetStandardAntennaGains(incidence_angles.hor_cbsd,
+               cbsd_grant.antenna_azimuth, cbsd_grant.antenna_beamwidth,
+               cbsd_grant.antenna_gain)
+
+  # Compute FSS antenna gain in the direction of CBSD
+  fss_ant_gain = antenna.GetFssAntennaGains(incidence_angles.hor_rx, 
+                   incidence_angles.ver_rx, fss_info.pointing_azimuth,
+                   fss_info.pointing_elevation, fss_info.max_gain_dbi)
+
+  # Get the total antenna gain by summing the antenna gains from CBSD to FSS
+  # and FSS to CBSD
+  effective_ant_gain = ant_gain + fss_ant_gain
+
+  # Compute the interference value for Fss co-channel entity
+  eirp = getEffectiveSystemEirp(max_eirp, cbsd_grant.antenna_gain,
+                   effective_ant_gain)
+  interference = eirp - db_loss - IN_BAND_INSERTION_LOSS
+  return interference
+
+
+def getFssMaskLoss(cbsd_grant, constraint):
+  """Gets the FSS mask loss for a FSS blocking protection constraint"""
+  # Get 50MHz offset below the lower edge of the FSS earth station
+  offset = constraint.low_frequency - 50.e6
+
+  # Get CBSD grant frequency range
+  cbsd_freq_range = cbsd_grant.high_frequency - cbsd_grant.low_frequency
+
+  fss_mask_loss = 0
+
+  # if lower edge of the FSS passband is less than CBSD grant
+  # lowFrequency and highFrequency
+  if constraint.low_frequency < cbsd_grant.low_frequency and\
```

Comment:
use parenthesis instead of \.
Here and below

Reply:
Updated   paranthesis